packages/fxa-auth-db-mysql/lib/db/schema/patch-108-109.sql

```diff
@@ -0,0 +1,97 @@
+-- This migration will help us to abstract the half baked recovery keys of fxa accounts. In this migration,
+-- we're adding a new `upsertRecoveryKey` procedure that will handle the half baked recovery keys by using
+-- the following criterias.
+
+-- 1. If a user has no recovery key then create a new recovery key.
+-- 2. If a user has unverified recovery key then delete that recovery key and create a new unverified
+--    recovery key.
+-- 3. If a user has verified recovery key then throw an exception
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('108');
+
+-- Delete these below 3 lines
```

Comment:
We typically don't have drops for procedures that were created in same migration.

Reply:
I forgot to remove that, that was for my testing purpose.